ipu-plugin/hack/cicd/check-go-fmt.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#Copyright (C) 2023 Intel Corporation
```

Comment:
2024

Reply:
These were old ones. I dont think we need to update them for each year